src/plugins/intel_gpu/src/graph/scatter_update.cpp

```diff
@@ -39,7 +39,16 @@ layout scatter_update_inst::calc_output_layout(scatter_update_node const& node)
     const size_t indices_size = node.input(1).get_output_layout().count();
     const size_t input_number_of_dims = node.input(0).get_output_layout().get_rank();
     const size_t updates_number_of_dims = node.input(2).get_output_layout().get_rank();
-    const size_t nonempty_indices_dims = GetNonEmptyDimsNumber(node.input(1).get_output_layout());
+
+    // convert axis for ie format
+    auto ie_axis = axis;
+    if (axis >= 2) {
+        auto spatial_axis = axis - 2;
+        const size_t default_dims = 4;
+        // Default and minimum number of dimensions is 4
+        auto spatial_size = std::max(input_number_of_dims, default_dims) - 2;
+        ie_axis = spatial_size - spatial_axis - 1 + 2;
```

Comment:
cant we pass ie_axis from ie?

Reply:
Fixed to align scatter update axis format with IE (https://github.com/andrew-k-park/openvino/commit/9f85459a839c29928d097fa6831aaa7c11ae8109)

Reply:
https://github.com/andrew-k-park/openvino/commit/9f85459a839c29928d097fa6831aaa7c11ae8109 Left quetion here

Reply:
Updated scatter_update primitive to propagate axis with integer type instead of `cldnn::scatter_update::scatter_update_axis` (https://github.com/andrew-k-park/openvino/commit/378db068783dca587b517bcdeee420a8cae33a79)